pkg/apis/core/validation/validation.go

```diff
@@ -3083,8 +3083,32 @@ func validateContainersOnlyForPod(containers []core.Container, fldPath *field.Pa
 	return allErrs
 }
 
+// PodValidationOptions contains the different settings for pod validation
+type PodValidationOptions struct {
+	// Allow pod spec to have more than one huge page resource (with different sizes)
+	AllowMultipleHugePageResources bool
+}
+
+// ValidatePodSingleHugePageResources
```

Comment:
complete godoc to describe what this does

Reply:
done. Thanks.